x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/index_permissions_callout.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { EuiCallOut, EuiLink } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { useMlKibana } from '../../../../contexts/kibana';
+
+export const IndexPermissionsCallout: FC<{ indexName: string; docsType: 'start' | 'create' }> = ({
+  indexName,
+  docsType,
+}) => {
+  const {
+    services: {
+      docLinks: {
+        links: {
+          ml: { dFAStartJob, dFACreateJob },
+        },
+      },
+    },
+  } = useMlKibana();
+
+  const docsLink = docsType === 'start' ? dFAStartJob : dFACreateJob;
+
+  return (
+    <EuiCallOut
+      title={i18n.translate('xpack.ml.dataframe.analytics.create.jobIdLabel', {
+        defaultMessage: 'You will not be able to create the job',
```

Comment:
I don't see the UI but if this message fits, I suggest to use this instead.
```suggestion
        defaultMessage: 'Job cannot be created',
```

Reply:
Updated in b087013f994faf2ebefa13d87a258c6ebdc34aa5